compiler/optimizer/Inliner.cpp

```diff
@@ -5011,6 +5011,18 @@ bool TR_InlinerBase::inlineCallTarget2(TR_CallStack * callStack, TR_CallTarget *
             _disableTailRecursion = true;
          }
       }
+   else if (comp()->getOption(TR_FullSpeedDebug) && tif->crossedBasicBlock())
```

Comment:
@ncough is working on just such a query now. He should be making his pull request soon so maybe he would be kind enough to sweep this test for FSD up into his new API when he asks to put it in? would that work for you @vijaysun-omr?